src/librustc_errors/lib.rs

```diff
@@ -638,7 +638,9 @@ impl Handler {
     }
 
     pub fn span_bug(&self, span: impl Into<MultiSpan>, msg: &str) -> ! {
-        self.inner.borrow_mut().span_bug(span, msg)
+        self.inner.borrow_mut().span_bug(span, msg);
+        // See 'bug()' for an more detailed explanation
```

Comment:
```suggestion
        // See `bug()` for a more detailed explanation.
```
The convention is to use backticks.